lib/routes/account.js

```diff
@@ -1113,13 +1115,21 @@ module.exports = function (
               push.notifyUpdate(uid, 'accountVerify')
 
               return db.verifyEmail(account)
-                .then(mailer.sendPostVerifyEmail.bind(
-                    mailer,
-                    account.email,
-                    {
-                      acceptLanguage: request.app.acceptLanguage
+                .then(
+                  function() {
+                    // Our post-verification email is very specific to sync,
+                    // so don't send it if we're sure this is not for sync.
+                    // Older clients will not send a 'service' param here
```

Comment:
There aren't many - in fact I'm pretty sure it's *only* content-server that calls this endpoint, because it's from a link in an email.  So the "older client" here would be an older version of content-server that we e.g. had to roll back for some unrelated reason.